bundler/lib/bundler/shared_helpers.rb

```diff
@@ -329,7 +329,7 @@ def set_path
     def set_rubyopt
       rubyopt = [ENV["RUBYOPT"]].compact
       setup_require = "-r#{File.expand_path("setup", __dir__)}"
-      return if !rubyopt.empty? && rubyopt.first =~ /#{Regexp.escape(setup_require)}/
+      return if !rubyopt.empty? && rubyopt.first.include?(setup_require)
```

Comment:
Not sure this is correct, there may be other stuff in RUBYOPT.

Reply:
matching against the regexp in this case is only checking if the string includes what's in the regexp, especially since the regexp is an escaped version of the string